content/variables/environment-variable-groups.md

```diff
@@ -84,14 +93,14 @@ Here you'll find some of the application environment groups and variables explai
         - google_play_credentials # <-- (GCLOUD_SERVICE_ACCOUNT_CREDENTIALS, GOOGLE_PLAY_TRACK)
         - other # <-- (PACKAGE_NAME, etc) 
 
-The above groups contain the following variables: 
+Add the above mentioned group environment variables in Codemagic UI (either in Application/Team variables):  
 
-        GCLOUD_SERVICE_ACCOUNT_CREDENTIALS: Encrypted(...) # <-- Put your encrypted Google Play service account credentials here.
+        GCLOUD_SERVICE_ACCOUNT_CREDENTIALS: # <-- Put your Google Play service account credentials here.
         FCI_KEYSTORE_PATH: /tmp/keystore.keystore
-        FCI_KEYSTORE: # <-- Put your encrypted keystore here
-        FCI_KEYSTORE_PASSWORD: # <-- Put your encrypted keystore password here
-        FCI_KEY_PASSWORD:  # <-- Put your encrypted key alias password here
-        FCI_KEY_ALIAS: # <-- Put your encrypted key alias here
+        FCI_KEYSTORE: # <-- Put your keystore here
+        FCI_KEYSTORE_PASSWORD: # <-- Put your keystore password here
+        FCI_KEY_PASSWORD:  # <-- Put your key alias password here
+        FCI_KEY_ALIAS: # <-- Put your key alias here
```

Comment:
I think we should change the format of this to indicate if the value needs to be base64 encoded. For example, the `FCI_KEYSTORE` needs to be encoded so we should mention it. 

Can we try making a table for these variables instead?